.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Thank you for contributing to the Firebase community!
+
+Have a usage question?
+=======================
+We get lots of those and we love helping you, but GitHub is not the best place for them and they will be closed. Please take a look at the guide first: https://firebase.google.com/docs/cli/
+
+If the official documentation doesn't help, try asking through our official support channel: https://firebase.google.com/support/
+
+Additional locations to check for solutions or assistance from the community:
+- Stack Overflow: https://stackoverflow.com/
+- Firebase Slack Community: https://firebase.community/
+
+*Please avoid duplicate posting across multiple channels!*
+
+Think you found a bug?
+=======================
+Yeah, we're definitely not perfect! Please use this template and include a minimal repro when opening the issue. If you know how to solve the issue, please create a Pull Request, and we'd be happy to review it!
+
+Have a feature request?
+========================
+Great, we love hearing how we can improve our products! However, do not use this template to submit a feature request. Please submit your feature requests to: https://firebase.google.com/support/contact/bugs-features/
+-->
+
+### [REQUIRED] Environment info
+
+<!-- What version of the Firebase CLI (`firebase-tools`) are you using? Note that your issue may already be fixed in the latest versions. The latest version can be found at https://github.com/firebase/firebase-tools/releases -->
+
+<!-- Output of `firebase --version` -->
+**firebase-tools:**
+
+<!-- e.g. macOS, Windows 10, Ubuntu 18.04 -->
```

Comment:
I wouldn't ask people for the specific platform versions (which version of Ubuntu, etc.). That's usually never relevant and takes extra work for people to find out.

Reply:
Good point. Interesting that I excluded the version from macOS anyway by instinct.